MAINTAINERS

```diff
@@ -508,3 +511,4 @@ M: Ira Cooper
 M: Shwetha Panduranga
 M: Nithya Balachandran
 M: Raghavendra Gowdappa
+M: Susant Palai
```

Comment:
Shouldn't this be M: Susant Palai?

Reply:
Please ignore this specific comment. I was looking only at DHT and missed that Susant was the Maintainer for Cloudsync.